src/common/components/announcement/index.tsx

```diff
@@ -0,0 +1,281 @@
+import React, { useState } from "react";
+import "./index.scss";
```

Comment:
add component folder into `styles/_components.scss` so you don't need to import file here.